### DISCUSSION TITLE
海外置產停看聽1／兩岸軍演+日幣貶 海外置產重起爐灶「詢問翻倍」

### DISCUSSION THREAD

**Discussion by Floriaand** [1101-GS4719-C/Python-Community-GS4719-C#1004] [General]
我覺得除了他說的幾個原因之外，還有就是台灣房價真的太高了，雖然政府有在打房，但是短期之內還是沒辦法看到很大的效果。我已經聽說過很多買房地產大賺的例子了，尤其是老一輩的很愛買房，很多都賺了好幾倍，很誇張。
網址：https://tw.news.yahoo.com/海外置產停看聽1-兩岸軍演-日幣貶-海外置產重起爐灶-詢問翻倍-220118404.html

**Comment by HuangHsinTzu** [comment]
這篇文章提到這波買房潮可能和緊張的台海關係有關
才讓我驚覺緊張的台海關係帶來的影響有多大
緊張的不只是對岸可能打過來的風險而是帶來的相關效應
另外，這篇文章也提醒一些海外投資房地產可能帶來的問題
希望投資人都真的能負擔這樣的風險，不要導致泡沫化的結果

**Comment by peggy0329** [comment]
身邊就有朋友很想去日本定居
一來是房價便宜，一來也是考慮未來台海關係
但是感覺目前這個議題都離我們還有一點距離：（
如果手上有資金，我也會考慮在海外置產
